components/Icons/Icons.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import UserLogo from 'static/images/icons/user.svg';
+import { string, bool } from 'prop-types';
+import styles from './Icons.module.css';
+
+Icon.propTypes = {
+  name: string.isRequired,
+  isNavIcon: bool,
+};
+
+function Icon({ name, isNavIcon }) {
```

Comment:
Can you please simply inline this icon instead of making an Icon component? I'm opposed to the idea of an Icon component.

Reply:
Idk how can i bypass this. I've made this because i can't pass an icon which is imported as a ReactComponent in props. What do u mean exactly by inline this. Having a file utils-icon with a function getIcon who returns jsx ?

Reply:
Good question! Here's what I mean:

Delete this `components/Icon` folder.

In NavListItem, you have an `icon` prop. Instead of accepting a string, you can accept an element. So, when you call `NavListItem`, you can pass it an icon!

```jsx
const Nav = (props) => {
  /* ... */
  return (
    {/* ... stuff ... */}
      <NavListItem
        {...otherProps}
        icon={otherProps.name === 'Account ' ? <UserLogo /> : null}
      />
    {/* ... stuff ... */}
  );
};

```

Reply:
Thanks for the reply, i've done something different but my last change are not correct i'll apply the code above instead ;)